app/mailers/project_mailer.rb

```diff
@@ -0,0 +1,10 @@
+class ProjectMailer < ApplicationMailer
+  layout 'mailer'
```

Comment:
Ah, I thought I had removed everything from `rails generate` but I had missed that line.